Update documentations: Readme, contributing and installation (#3563)
vector-im/element-ios: README.md

[![Twitter URL](https://img.shields.io/twitter/url?label=Element&url=https%3A%2F%2Ftwitter.com%2Felement_hq)](https://twitter.com/element_hq)

Element iOS is an iOS [Matrix](https://matrix.org/) client provided by [Element](https://element.io/). It is based on [MatrixKit](https://github.com/matrix-org/matrix-ios-kit) and [MatrixSDK](https://github.com/matrix-org/matrix-ios-sdk).

<p align="center">  
  <a href=https://itunes.apple.com/us/app/element/id1083446067?mt=8>
  <img alt="Download on the app store" src="https://linkmaker.itunes.apple.com/images/badges/en-us/badge_appstore-lrg.svg" width=160>
  </a>
</p>

## Beta testing 

You can try last beta build by accessing our [TestFlight Public Link](https://testflight.apple.com/join/lCeTuDKM). For questions and feedback about latest TestFlight build, please access the [Element iOS Matrix room](#element-ios:matrix.org).

## Build instructions

To build the application please refer to the [installation guide](./INSTALL.md).

## Contributing

If you want to contribute to Element iOS code or translations please refer to the [contribution guide](CONTRIBUTING.md).

## Support

> When you are experiencing an issue on Element iOS. Please first search in [GitHub issues](https://github.com/vector-im/element-ios/issues)

Reviewer: ```suggestion
When you are experiencing an issue on Element iOS, please first search in [GitHub issues](https://github.com/vector-im/element-ios/issues)
```

> and then in [Element iOS Matrix chat room](#element-ios:matrix.org).

Reviewer 2: This text does not render the link. Try:

```suggestion
and then in Element iOS Matrix chat room ([#element-ios:matrix.org](https://matrix.to/#/#element-ios:matrix.org)).
```

> If after your research you still have a question, ask at our [Matrix chat room](#element-ios:matrix.org). Otherwise feel free to create a GitHub issue if you encounter a bug or a crash, by explaining clearly in detail what happened. You can also perform bug reporting (Rageshake) from the Element application by shaking your phone or going to the application settings. This is especially recommended when you encounter a crash.

Reviewer 2: ```suggestion
If after your research you still have a question, ask at [#element-ios:matrix.org](https://matrix.to/#/#element-ios:matrix.org). Otherwise feel free to create a GitHub issue if you encounter a bug or a crash, by explaining clearly in detail what happened. You can also perform bug reporting (Rageshake) from the Element application by shaking your phone or going to the application settings. This is especially recommended when you encounter a crash.
```

## Copyright & License

> Copyright (c) 2014-2017 OpenMarket Ltd Copyright (c) 2017 Vector Creations Ltd Copyright (c) 2017-2019 New Vector Ltd

Reviewer 2: ```suggestion
Copyright (c) 2014-2017 OpenMarket Ltd
Copyright (c) 2017 Vector Creations Ltd
Copyright (c) 2017-2020 New Vector Ltd
```
